components/ChildCard.jsx

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import React from "react";
+import styled from "styled-components";
+
+const StyledChildCard = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  width: 80%;
+  min-height: 150px;
+  padding: 10px;
+  overflow: hidden;
+
+  border-radius: 20px;
+  background: #fff;
+  box-shadow: 4px 4px 4px 0px rgba(0, 0, 0, 0.1);
+`;
+
+const StyledImage = styled.div`
```

Comment:
This components name is not ideal, it is rather an ImageWrapper than the StyledImage itself:

```suggestion
const ImageWrapper = styled.div`
```